python/python/glide/async_commands/redis_modules/json/commands.py

```diff
@@ -0,0 +1,109 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import json
+from typing import Dict, List, Optional, Union, cast
+
+from glide.async_commands.core import ConditionalChange
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+TJson = Union[str, int, float, bool, None, List["TJson"], Dict[str, "TJson"]]
+
+
+class Json:
+    """Class for RedisJson commands."""
+
+    async def set(
+        self,
+        client: TRedisClient,
+        key: str,
+        path: str,
+        value: TJson,
+        set_condition: Optional[ConditionalChange] = None,
+    ) -> Optional[TOK]:
+        """
+        Sets the JSON value at the specified `path` stored at `key`.
+        See https://redis.io/commands/json.set/ for more details.
+
+        Args:
+            client (TRedisClient): The Redis client to execute the command.
+            key (str): The key of the JSON document.
+            path (str): Represents the path within the JSON document where the value will be set.
+                The key will be modified only if `value` is added as the last child in the specified `path`, or if the specified `path` acts as the parent of a new child being added.
+            value (TJSON): The value to set at the specific path.
+            set_condition (Optional[ConditionalChange]): Set the value only if the given condition is met (within the key or path).
+                Equivalent to [`XX` | `NX`] in the Redis API. Defaults to None.
+
+        Returns:
+            Optional[TOK]: If the value is successfully set, return OK.
+                If value isn't set because of `set_condition`, return None.
+
+        Examples:
+            >>> await json_set(client, "doc", "$", {'a': 1.0, 'b': 2})
+                'OK'  # Indicates successful setting of the value at path '$' in the key stored at 'doc'.
+            >>> await json_set(client, "doc", "$.a", 1.5 , ConditionalChange.ONLY_IF_DOES_NOT_EXIST)
+                None  # Indicates that the value at path '$.a' in the key stored at 'doc' was not set because the key already exists.
+        """
+        args = [key, path]
+        args.append(json.dumps(value))
+        if set_condition:
+            args.append(set_condition.value)
+
+        return cast(
+            Optional[TOK], await client._execute_command(RequestType.JsonSet, args)
+        )
+
+    async def get(
+        self,
+        client: TRedisClient,
+        key: str,
+        paths: Optional[Union[str, List[str]]] = None,
+        indent: Optional[str] = None,
+        newline: Optional[str] = None,
+        space: Optional[str] = None,
```

Comment:
Do you want to merge this into `JsonOptions`?

Reply:
Regarding Yuri's proposal, are those options being utilized in other JSON functions as well?